sourcegraph/learn
# Add content: How to integrate Sourcegraph into your workflows

## Type of change


### What should this PR do?

Add content: [DEVED-116](https://sourcegraph.atlassian.net/browse/DEVED-116)

### What are the acceptance criteria? 

- The page displays correctly at the URL
- The content card displays correctly on the index page
- The embedded video loads correctly
- The header/thumbnail loads correctly
- The front-matter is accurate and complete

### How should this PR be tested?

- Open the deploy preview
- Check the above criteria

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
